src/Microsoft.PowerShell.Security/security/AclCommands.cs

```diff
@@ -1144,7 +1141,7 @@ private IntPtr GetSaclWithCapId(string capStr)
                     }
 
                     // Find the supplied string among available CAP names, use the corresponding CAPID.
-                    NativeMethods.CENTRAL_ACCESS_POLICY cap = new NativeMethods.CENTRAL_ACCESS_POLICY();
+                    NativeMethods.CENTRAL_ACCESS_POLICY cap;
```

Comment:
I think this can be removed here and instead be defined in the for loop body scope, where it is only used.